applications/PGSAUT-abstract.md

```diff
@@ -0,0 +1,53 @@
+# Abstract for The PGSA@UT Philosophy Conference
+
+### Abstract for submission to The [Philosophy Graduate Student Association at The University of Tennessee Philosophy Conference](http://utkphilgradconference.weebly.com/) March 13-14 2021
+
+## Hacking's Human Kinds and the Ethics of Participatory Science 
+
+Drawing on Hacking’s ([1996](doi.org/\10.1093\acprof:oso\9780198524021.001.0001) influential concept of the ‘looping effects of human kinds’, this paper will explore the ethical implications of the dynamics between researchers and research subjects in scientific practice. 
+We argue that scientists engaged in work about human kinds have a moral duty to include research subjects in their work. 
+We will then ground the theoretical discussion with a case study of participatory science in action: a project to build a tool to investigate autism and sensory processing through the participatory co-creation of a citizen science platform. 
+This project is a collaboration between the research charity Autistica and The Alan Turing Institute, the UK’s national institute for AI and data science. 
```

Comment:
```suggestion
This project is a collaboration between the UK-based research charity Autistica and The Alan Turing Institute, the UK’s national institute for AI and data science. 
```

Reply:
If it is UK-based, suggest specifying this for an international audience (if it isn't, ignore this!)